sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/RspecStrings.resx

```diff
@@ -7291,7 +7291,7 @@
     <value>15min</value>
   </data>
   <data name="S4018_Severity" xml:space="preserve">
-    <value>Major</value>
+    <value>Minor</value>

```

Comment:
Some of the properties in resx changed, so we need to either revert them (and corresponding json changes) or have another release of SonarC#.

Both options are fine, which one do you think is less painful?